pages/listings/listing/create/index.html

```diff
@@ -33,15 +33,16 @@ <h1 class="text-center mt-4">Create New Listing</h1>
               </div>
               <div class="form mb-3">
                 <label for="location">Location</label>
-                <input class="form-control bg-theme-light" style="border-radius: 0px" type="text" name="location" id="createLocation" data-auth="createFormLocation" required minlength="3" maxlength="20" />
+                <input class="form-control bg-theme-light" style="border-radius: 0px" type="text" name="location" id="createLocation" data-auth="createFormLocation" required minlength="3" maxlength="20" aria-label="Put in your location" />
               </div>
               <div class="form mb-3">
                 <label for="deadline">Application Deadline</label>
-                <input class="form-control bg-theme-light" style="border-radius: 0px" type="datetime-local" name="deadline" id="createDeadline" data-auth="createFormLocation" required title="Enter a valid date" />
+                <input class="form-control bg-theme-light" style="border-radius: 0px" type="datetime-local" name="deadline" id="createDeadline" data-auth="createFormLocation" required title="Enter a valid date"  aria-label="Deadline date" />
+
               </div>
               <div class="input-group mt-5 mb-3">
                 <span class="input-group-text" style="border-radius: 0px">Upload Cover Picture</span>
-                <input type="text" class="form-control bg-theme-light" style="border-radius: 0px" id="pictureUrl" data-auth="createFormPicture" />
+                <input type="text" class="form-control bg-theme-light" style="border-radius: 0px" id="pictureUrl" data-auth="createFormPicture" aria-label="Upload cover picture" />
```

Comment:
Ensure label accurately reflects the users input - consider "Enter URL for cover picture"